src/coreclr/nativeaot/System.Private.CoreLib/src/System/EETypePtr.cs

```diff
@@ -10,17 +10,15 @@
 **
 ===========================================================*/
 
-using System.Runtime;
 using System.Diagnostics;
-using System.Runtime.InteropServices;
+using System.Runtime;
 using System.Runtime.CompilerServices;
-
+using System.Runtime.InteropServices;
 using Internal.Runtime.CompilerServices;
-
+using CorElementType = System.Reflection.CorElementType;
```

Comment:
This is deleting newlines between `using X` and `using Foo = X.Foo`. The repo style disallows separating these with newlines?

We also separate System and Internal namespace with newlines. Is the repo style disallowing that too?

Reply:
Right now it does, yeah. There is an option to separate using groups with a newline: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/dotnet-formatting-options#dotnet_separate_import_directive_groups

I can set that for the NativeAOT sources paths, we don't seem to do that in other parts of the repo.

Reply:
@MichalStrehovsky should look better now